module-services/service-db/DBServiceAPI.cpp

```diff
@@ -26,6 +26,12 @@
 #include <utility>
 #include <cassert>
 
+namespace constants
+{
+    inline constexpr std::uint32_t DefaultTimeoutInMs = 10000U;
+    inline constexpr std::uint32_t BackupTimeoutInMs  = 120000U;
```

Comment:
`inline` is not needed when defining variables in .cpp files.

Reply:
done